### PR TITLE
Add Reading Council (reading.gov.uk) source

### PR DESCRIPTION
**Pull request by nashant**
Added source for Reading Council

```
waste_collection_schedule/test/test_sources.py -s reading_gov_uk
Testing source reading_gov_uk ...
  found 10 entries for known_uprn
  found 10 entries for known_uprn as number
  found 10 entries for unknown_uprn_by_number
```

### CODE REVIEW

File: doc/source/reading_gov_uk.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      args:
+        uprn: UPRN
+```
+
+### Configuration Variables
+
+**postcode**  
+_(string) (optional)_
+
+**hournameornumber**
```
[the remark this answers is not in the document]
Comment (nashant) [reply doc/source/reading_gov_uk.md]: Fixed, and in 2 other docs too

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/reading_gov_uk.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+SEARCH_URLS = {
+    "UPRN": "https://api.reading.gov.uk/rbc/getaddresses",
+    "COLLECTION": "https://api.reading.gov.uk/api/collections"
+}
+
+
+class Source:
+    def __init__(
+        self, uprn=None, postcode=None, housenumberorname=None
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/reading_gov_uk.py:39]: `housenumberorname` is not the same as defined in the md file, so example configuration provided by md file would not work

Comment (nashant) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/reading_gov_uk.py]: Fixed

File: doc/source/cornwall_gov_uk.md [hunk 3]
```diff
@@ -17,13 +17,13 @@ waste_collection_schedule:
 
 ### Configuration Variables
 
-**postcode**  
+**postcode**
 *(string) (optional)*
 
-**hournameornumber**  
+**housenameornumber**
```
Comment (5ila5) [inline doc/source/cornwall_gov_uk.md:23]: Not the same as.
https://github.com/mampfes/hacs_waste_collection_schedule/blob/1f53eaf87b22de8778deaa202ae3a3455eefac41/custom_components/waste_collection_schedule/waste_collection_schedule/source/cornwall_gov_uk.py#L22-L25
 Would be nice if you could rename this to  housenumberorname (here and in yaml parts). changeing the python might break existing configurations

Comment (nashant) [reply doc/source/cornwall_gov_uk.md]: Done

File: doc/source/westberks_gov_uk.md [hunk 4]
```diff
@@ -17,13 +17,13 @@ waste_collection_schedule:
 
 ### Configuration Variables
 
-**postcode**  
+**postcode**
 _(string) (optional)_
 
-**hournameornumber**
```
[the remark this answers is not in the document]
Comment (nashant) [reply doc/source/westberks_gov_uk.md:23]: And done

**Comment by marktay** [comment]
Hi nashant - cannot see this one in the source dir after a fresh install from HACS

**Comment by 5ila5** [comment]
> Hi nashant - cannot see this one in the source dir after a fresh install from HACS

It's not yet part of a release it will be part of version 1.40.0. Or can be installed now as the git/master version.
Installing it with HACS istalls the newest release (1.39.0) by default.

**Pull request merged**